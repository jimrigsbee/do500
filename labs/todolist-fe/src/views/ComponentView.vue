<template>
  <div class="home">
    <h2>Overview of the todo app's component structure</h2>
    <img src="@/assets/ComponentsLabs.png"/>
  </div>

</template>

<script>
export default {
  name: "ComponentView",
  data() {
    return {};
  }
};
</script>
