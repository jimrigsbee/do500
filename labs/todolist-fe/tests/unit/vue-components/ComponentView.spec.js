/* eslint-disable */
import { shallow } from "@vue/test-utils";
import ComponentView from "@/views/ComponentView.vue";
import * as all from "../setup.js";

describe("ComponentView.vue", () => {
  it("should render like the snapshot", () => {
    const wrapper = shallow(ComponentView);
    expect(wrapper.element).toMatchSnapshot();
  });
});
