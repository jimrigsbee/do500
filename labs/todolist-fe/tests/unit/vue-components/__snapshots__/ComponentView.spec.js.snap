// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentView.vue should render like the snapshot 1`] = `
<div
  class="home"
>
  <h2>
    Overview of the todo app's component structure
  </h2>
   
  <img
    src="@/assets/ComponentsLabs.png"
  />
</div>
`;
